src/components/kpis-bar/kpis-bar.less

```diff
@@ -83,52 +64,99 @@
       }
       @media screen and (max-width: 675px) {
         top: -35px;
-        right: 20px;
+        right: 67px;
         bottom: 0;
       }
     }
   }
 
-  .title {
-    .text-center {
-      line-height: 3;
-      font-variant: small-caps;
-      font-size: larger;
+
+  .content {
+    padding: 15px;
+    min-height: @kpi-max-height + 30px;
+    // overflow: hidden;
+    background-color: @kpis-bar-background-color;
+    box-shadow: @kpis-bar-box-shadow;
+
+    &.empty { min-height: 50px; }
+
+    transition: all .2s ease-in;
+    @media screen and (min-width: 725px) {
+      padding: 0 60px;
     }
-    .badge {
-      position: absolute;
-      top: 8px;
-      left: -10px;
-      background-color: #d1e55c;
-      color: #4f5959;
-      box-shadow: 0px 1px 3px -1px;
+    @media screen and (max-width: 675px) {
+      margin-top: 18px;
     }
-    .no-kpi-templates {
-      color: #afadb5;
-      font-weight: 500;
+    .impac-kpi {
+      .column-gutter(5px);
+
+      @media screen and (min-width: 1200px) {
+        padding-left: 20px;
+        padding-right: 20px;
+      }
+
+      .kpi.add {
+        .kpi-show .caption {
+          white-space: inherit;
+        }
+      }
     }
-  }
 
-  .add-bar {
-    border-bottom: dashed 1px;
-  }
+    &.hide-animate {
+      min-height: 0;
+    }
 
-  .kpi-loader {
-    display: flex;
-    align-items: center;
-    justify-content: center;
-    min-height: @kpi-max-height + 20px;
-  }
 
-  .dates-picker-container {
-    width: 40%;
-    margin: 0 auto;
+    .title {
+      .text-center {
+        line-height: 3;
+        font-variant: small-caps;
+        font-size: larger;
+      }
+      .badge {
+        position: absolute;
+        top: 8px;
+        left: -10px;
+        background-color: #d1e55c;
+        color: #4f5959;
+        box-shadow: 0px 1px 3px -1px;
+      }
+      .no-kpi-templates {
+        color: #afadb5;
+        font-weight: 500;
+      }
+    }
 
-    .sdp-from-label,
-    .sdp-to-label {
-      text-transform: uppercase;
-      font-weight: 400;
+    .add-bar {
+      border-bottom: dashed 1px;
     }
+
+    .kpi-loader {
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      min-height: @kpi-max-height + 20px;
+    }
+
+    .dates-picker-container {
+      width: 40%;
+      margin: 0 auto;
+
+      .sdp-from-label,
+      .sdp-to-label {
+        text-transform: uppercase;
+        font-weight: 400;
+      }
+    }
+  }
+
+  .show-animate {
+    transition: opacity 1s ease-in;
+    opacity: 1;
```

Comment:
Opacity (and especially opacity alone) is not the best choice here... Im not sure if you have noticed, but once you hide the kpis-bar you can still click on the buttons even though they are not visible, they are still in the DOM. The whole kpis-bar and kpis, and buttons are still in the dom. 
This is obviously not acceptable as there is a big grey space between the widgets and the top of the page, of which, users can still interact with. 

1. Can you please fix this with I think one of two options:
   1. by using a slide up for hide, and slide down for show animation instead of the opacity. As we want the kpis-bar empty space to reduce as if the kpis-bar was not there,
   2. or, after the opacity finishes fading, then using transition to reduce / expand the height of the kpis bar. (if you use both opacity and height transition, increase the speed of the opacity fade on the kpis).

2. I think you should use a JS method for more control, remember you can apply ngShow/ngHide with the css class `'hide'` & `'show'`. This removes it from the DOM. So maybe you could do something like `angular.element(el).animate(...)` then  `$timout(-> element.addClass('hide'), durationOfSlideAnimation`.

Take a look at kpi.directive.coffee `animateKpiPanels` method for inspiration.

Make sure when in hidden mode, the KPIs and kpis-bar buttons are not visible or interactable.